mod/deckorate_attribution/assets/style/reference.scss

```diff
@@ -64,6 +60,36 @@
   }
 }
 
+.TYPE-reference {
```

Comment:
I suspect if you open the reference card in `:board` (the "Advanced" view), there will be other submit buttons and card forms within the same set. 

maybe `TYPE-reference > .card-editor` would be safe?